### PR TITLE
Starting to move out of API menu

### PR DESCRIPTION
**Pull request by AdrianGjerstad**
Began the migration of information into a new menu, which can be seen by clicking on the "CookieAuto Interface" button in the milk.

begins #30

### CODE REVIEW

File: auto.js [hunk 1]
```diff
@@ -563,34 +569,120 @@ var CookieAuto = {};
...
+                    this.menuelt.style.height = "calc(100vh - 144px)";
+                    this.menuelt.style.wordWrap = "break-word";
+                    this.menuelt.style.zIndex = "10";
+                    this.menuelt.style.background = "#000 url(img/darkNoise.jpg)";
+                    this.menuelt.style.overflow = "auto";
+                    this.menuelt.style.display = "none";
+
+                    document.body.appendChild(this.menuelt);
+
+                    Game.prefs.buyscript_abbuild = CookieAuto.control.buyBuildings?1:0;
```
Comment (lmgjerstad) [inline auto.js]: I know we need CookieAuto references in HTML, but for cases like this, replace CookieAuto.control with settings

File: auto.js [hunk 2]
```diff
@@ -478,6 +478,10 @@ var CookieAuto = {};
 
                 CookieAuto.updateGoalValue();
 
+                if (Game.onMenu !== "" && CookieAuto.ui.showing) {
+                    CookieAuto.ui.showMenu.call(CookieAuto.ui);
```
Comment (lmgjerstad) [inline auto.js]: why the complicated syntax?  Shouldn't this just be CookieAuto.ui.showMenu? 

Better yet, I'm not sure why ui was actually put inside CookieAuto.

File: auto.js [hunk 3]
```diff
@@ -488,6 +492,8 @@ var CookieAuto = {};
 
                 this.shoppingListSortMode = this.sorting.price;
 
+                this.ui.init.call(this.ui);
```
Comment (lmgjerstad) [inline auto.js]: Another instance of using call where it doesn't do anything more than just calling the function directly

File: auto.js [hunk 4]
```diff
@@ -563,34 +569,120 @@ var CookieAuto = {};
...
+                    res += '<div id="shlst'+o.id+'" class="icon" onclick="CookieAuto.toggleInShoppingList(Game.UpgradesById['+o.id+']);" onmouseout="Game.setOnCrate(0);Game.tooltip.shouldHide=1;" onmouseover="if (!Game.mouseDown) {Game.setOnCrate(this);Game.tooltip.dynamic=1;Game.tooltip.draw(this,function(){return function(){return Game.crateTooltip(Game.UpgradesById['+o.id+'],\'shoppingListSelector\');}();},\'\');Game.tooltip.wobble();}" style="padding:0; cursor:pointer; background-position:'+x+'px '+y+'px; float:left; opacity:'+(inShoppingList(o)?'1':'0.2')+';"></div>';
                 }
                 return res;
+            },
+
+            ui : {
+                showing : false,
+                menuelt : null,
```
Comment (lmgjerstad) [inline auto.js]: I have no idea what this variable name means, can you choose a more descriptive name?

**Review by lmgjerstad — CHANGES_REQUESTED** [review]

File: auto.js (hunk 1 above)
Comment (AdrianGjerstad) [reply auto.js]: done

File: auto.js (hunk 2 above)
Comment (AdrianGjerstad) [reply auto.js]: done

File: auto.js (hunk 3 above)
Comment (AdrianGjerstad) [reply auto.js]: done

File: auto.js (hunk 4 above)
Comment (AdrianGjerstad) [reply auto.js]: done

**Review by lmgjerstad — APPROVED** [review]

**Pull request merged**